instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -325,58 +325,6 @@ CreateRhelPackage()
     	${repoDir}/zm-build/${currentScript}.spec
     echo "%attr(644, zimbra, zimbra) /opt/zimbra/conf/*" >> \
     	${repoDir}/zm-build/${currentScript}.spec
-    echo "%attr(755, zimbra, zimbra) /opt/zimbra/conf/templates" >> \
```

Comment:
just curious
does anybody know why we were copying templates to /opt/zimbra/conf directory?

Reply:
@silentsakky I surely do not!

Reply:
@silentsakky , I have same question. We badly need someone to go through webclient/admin console code or history and find it out.

Reply:
@Prashantsurana https://bugzilla.zimbra.com/show_bug.cgi?id=63368